.github/workflows/helm-weekly-release-pr.yaml

```
@@ -0,0 +1,30 @@
+name: helm-weekly-release-pr
+
+on:
+  schedule:
+    - cron: '0 12 * * 1' # 12 UTC on Mondays for weekly release; images for GEM and Mimir should have already been pushed
```

Needs to run on workdays if we missed the image generation on Monday. Must be idempotent since it may be run by manually.

```suggestion
    - cron: '0 12 * * 1-5' # 12 UTC on workdays for weekly release; images for GEM and Mimir should have already been pushed
```